pages/apis/rest_api/analytics/flaky_tests.md.erb

```diff
@@ -0,0 +1,52 @@
+# Flaky tests API
+The Flaky test API endpoint provides information about tests detected as flaky in a test suite.
+
+<section class="callout callout--troubleshooting">
+  <p class="callout__title" id="a-callout-title">🚧 Beta feature</p>
+  <p>Flaky test detection is in beta and not available to all organizations.</p>
+</section>
```

Comment:
I think we can remove this now that the _Beta_ label is more apparent on the page.